qemu/tests/block_resize.py

```diff
@@ -1,14 +1,16 @@
 import logging
 import re
 
-from autotest.client.shared import error
```

Comment:
We need consider compatibility problem, you can refer:
https://github.com/autotest/tp-qemu/commit/6f7b59fa5b6f4dbec4af4c2d2e20d31848d684a4